packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Documents.Example.tsx

```diff
@@ -203,25 +203,39 @@ export class DetailsListDocumentsExample extends React.Component<{}, IDetailsLis
         </div>
         <div className={classNames.selectionDetails}>{selectionDetails}</div>
         {announcedMessage ? <Announced message={announcedMessage} /> : undefined}
-        <MarqueeSelection selection={this._selection}>
+        {isModalSelection ? (
+          <MarqueeSelection selection={this._selection}>
+            <DetailsList
+              items={items}
+              compact={isCompactMode}
+              columns={columns}
+              selectionMode={SelectionMode.multiple}
+              getKey={this._getKey}
+              setKey="multiple"
+              layoutMode={DetailsListLayoutMode.justified}
+              isHeaderVisible={true}
+              selection={this._selection}
+              selectionPreservedOnEmptyClick={true}
+              onItemInvoked={this._onItemInvoked}
+              enterModalSelectionOnTouch={true}
+              ariaLabelForSelectionColumn="Toggle selection"
+              ariaLabelForSelectAllCheckbox="Toggle selection for all items"
+              checkButtonAriaLabel="Row checkbox"
+            />
+          </MarqueeSelection>
+        ) : (
           <DetailsList
             items={items}
             compact={isCompactMode}
             columns={columns}
-            selectionMode={isModalSelection ? SelectionMode.multiple : SelectionMode.none}
+            selectionMode={SelectionMode.none}
             getKey={this._getKey}
-            setKey="set"
+            setKey="none"
```

Comment:
What does changing this from `set` to `none` do? Does set automatically choose the first item or something like that as a default?

Reply:
It's just a key for the list! The value doesn't matter so much, but I thought I'd label them based on the scenarios they're demonstrating.